### ISSUE TITLE
Add proofs

### ISSUE THREAD

**Issue by GrimPixel** [jdm-contrib/jdm#280]
I see some sites have changed their policies during time. So the information can be slightly outdated. If there are links to the websites' privacy policy or “How to delete my account” page, then there would be fewer misunderstandings.

**Comment by tupaschoal** [comment]
We try to provide links to "how to delete page" as often as possible, but privacy policy ones we don't. They can also change, I don't think it would help that much, maybe I've miscomprehended

**Comment by GrimPixel** [comment]
OK, I think having links to the latest “How to delete my account” pages is enough. Privacy policy is for https://tosdr.org/.

I think there can be a “contribute” button just like that site.

I think you can add your site on https://www.privacytools.io/. I had trouble when creating pull request.

**Comment by tupaschoal** [comment]
Based on a comment in issue #170, it is difficult for us to implement and maintain this kind of user interactive interface.

Would supporting the privacy policy field without this still be valuable, in your opinion?

**Comment by GrimPixel** [comment]
Well, in my opinion, the name of this site suggests that it tells people how to delete their accounts. If the entire privacy policy is to be discussed, it would be too big. There is already a ToSDR doing this. I think JustDeleteMe may focus on deleting accounts so it can provide information of more websites.

If it is too hard to implement that button, I think there may be a feedback form to tell you about the information of deleting accounts. Because not every one who wants to contribute knows about how to fork on Github.

**Comment by tupaschoal** [comment]
Yeah, I know it is not straight forward to fork, for people which are not Github-savvy. Using a button or a form kind of amounts to the same, as our website is completely read only, unless you're talking about hosting the form somewhere else.

**Comment by GrimPixel** [comment]
What about using EtherCalc for contribution?

**Comment by tupaschoal** [comment]
I never thought about it. We often populate data with info that people provide on issues, which can kInd of work like a feedback form, if the user is a little curious.
@Nightfirecat and @PyroSamurai what do you think?
